src/lib/containers/TermsAndConditions.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useEffect, useState } from 'react'
+import { tcItem } from './TermsAndConditionsItem'
+import TermsAndConditionsItem from './TermsAndConditionsItem'
+import { ReactComponent as chatBubblesSvg } from '../assets/icons/terms/chat-bubbles.svg'
+import { ReactComponent as lockSvg } from '../assets/icons/terms/lock.svg'
+import { ReactComponent as scaleSvg } from '../assets/icons/terms/scale.svg'
+import { ReactComponent as shieldSvg } from '../assets/icons/terms/shield.svg'
+import { ReactComponent as peopleSvg } from '../assets/icons/terms/people.svg'
+import { ReactComponent as penSvg } from '../assets/icons/terms/pen.svg'
+import { ReactComponent as speakerSvg } from '../assets/icons/terms/speaker.svg'
+import { ReactComponent as flagSvg } from '../assets/icons/terms/flag.svg'
+import { ReactComponent as RefreshSvg } from '../assets/icons/terms/refresh.svg'
+import { Button, Modal } from 'react-bootstrap'
+import SignatureCanvas from 'react-signature-canvas'
+import ReactSignatureCanvas from 'react-signature-canvas'
+
+export type TermsAndConditionsProps = {
+  token?: string
```

Comment:
I'm having a difficult time understanding the contract between the outer page and this component.  
I don't think you need the session token.
But I think you need to add a callback function in the props that returns a boolean (you would call on every change event, to let the caller know if all items have been consented to).   Then the outer page can set up the enable states based on this value.

Reply:
I updated the code to use a callback function to send back a boolean to the caller. 

```
export type TermsAndConditionsProps = {
  onFormChange: Function
}
```